news/2020-31-12-mappers-report-fall.md

```diff
@@ -0,0 +1,265 @@
+
+We hope to see MJHs make more maps in the future as it’s pretty clear he’s making a name for himself in the 5K community thus far, and hopefully we can see him rank some maps whilst he’s at it!
+
+## <a id="mapping-contests"></a>Mapping Contests
+
+Did someone ask for contests? No? Ahh well, you get them anyways! As always we inform you about any interesting mapping related contests going on. Even if you do not plan to participate in them, you might want to check out if any of them has an open voting phase going on! Don't forget: Every vote counts!
+
+And don't forget. Should you have any interesting contests planned or know about, let us know! We would love to give them more attention in our news post!
+
+### Monthly Beatmapping Contest
+
+The Monthly Beatmapping Contest series is continuing as strongly as ever, albeit with some changes, but also with some awesome results to share! Starting with MBC \#8, the contest has changed to a judge-based format. Where before screeners would pick maps for the community to vote on, now screeners function the same but the final vote comes down to a panel of judges with detailed impressions and scoring. For MBC \#9 the formula was further mixed up by making the contest have a **theme** instead of a **restriction**. With these changes, we get to see an ever-evolving regular contest format to stay excited about!
+
+For MBC \#8 the restriction was that sliders could not be used, and [Mao](https://osu.ppy.sh/users/2204515)'s entry of [Silentroom - NULCTRL](https://osu.ppy.sh/beatmapsets/1266008#osu/2631203) with its excellent use of circles, **circle size 7**, and screen-usage based patterning won over the judge's hearts. He has now earned enough points in MBC for a tier 2 profile badge, and is all that much closer to attaining the coveted title of elite mapper! Well deserved!
+
+In MBC \#9 the theme was that every object must overlap with another one. With its results mapper [xbopost](https://osu.ppy.sh/users/6842421) came in first place with his entry of [PUP - See You At Your Funeral](https://osu.ppy.sh/beatmapsets/1289138#osu/2676185) which took the theme well to heart and built a unique and interesting map and gameplay experience that you should be sure to check out! That's not all though, as this is xboposts' **third** time coming in first place in MBC and they have now earned the esteemed title of **Elite Mapper**!
```

Comment:
needs commas at "results" and "after the map link"

took the theme well to heart ~~and~~ **as they [he/she]** built a unique and interesting